pages/glossary/decorative.md

```diff
@@ -3,4 +3,6 @@ title: Decorative
 key: decorative
 ---
 
-Serving only an aesthetic purpose, providing no information, and having no functionality.
\ No newline at end of file
+Serving only an aesthetic purpose, providing no information, and having no functionality. 
+
+**Note:** Authors can mark an `img` element as decorative to indicate that it should be ignored by assistive technology by using either `role="presentation"`, `role="none"`, or `alt=""`. An element should only be marked as decorative if removing the element does not cause a loss of information to the user.
```

Comment:
Could this also apply to elements other than 'img', such as 'svg'?

Reply:
The context of this rule is `img` so the note addresses that specifically and not other elements.

Reply:
I could change this to say "Authors can mark elements as decorative..." if you think that would make more sense?